pkg/collector/corechecks/cluster/ksm/kubernetes_state_transformers_test.go

```diff
@@ -487,6 +487,7 @@ func Test_jobStatusFailedTransformer(t *testing.T) {
 					Labels: map[string]string{
 						"job_name":  "foo-1509998340",
 						"namespace": "default",
+						"reason":    "BackoffLimitExceeded",
```

Comment:
Should these reasons be in `tags` instead of `Labels` ?

Reply:
I think it’s correct:
`reason:BackOffLimitExceeded` is a label that KSM attaches on `kube_job_status_failed` KSM metric and that we want to drop in the Datadog `kubernetes_state.job.failed` metric.
That’s why no corresponding DD tag has been added for this KSM label.

Reply:
I see :+1: , but then there isn't a test that exercises the added code right? Because for that, the `reason:` prefix should be in a tag, and not a label.